js/src/Ice/Connection.js

```diff
@@ -71,8 +120,12 @@ export const [HeaderDict, HeaderDictHelper] = defineDictionary(
  *  Provides access to the connection details of a WebSocket connection
  **/
 export class WSConnectionInfo extends ConnectionInfo {
```

Comment:
I suspect we don't need the defineDictionary above.

Reply:
Removed.